Providers/Scripts/3.x/Scripts/nxAvailableUpdates.py

```diff
@@ -81,38 +88,53 @@ def GetAptUpdates(Name):
     code, out = RunGetOutput(cmd, False, False)
     #Get Security Patches list
     security_patch_list = []
-    code, out = RunGetOutput(prep_security_sources_list_cmd,False,False)
-    security_updates_cmd = dist_upgrade_simulation_cmd_template.replace('<SOURCES>', '-oDir::Etc::Sourcelist=' + security_sources_list)
-    code,out = RunGetOutput(security_updates_cmd,False,False)
-    srch_txt = r'Inst[ ](.*?)[ ].*?[(](.*?)[ ](.*?)[ ]\[(.*?)\]'
-    srch = re.compile(srch_txt, re.M | re.S)
-    pkg_list = srch.findall(str(out))
-    for package in pkg_list:
-        security_patch_list.append(package[0])
-    cmd = 'LANG=en_US.UTF8 apt-get -s dist-upgrade | grep "^Inst"'
-    LG().Log('DEBUG', "Retrieving update package list using cmd:" + cmd)
-    code, out = RunGetOutput(cmd, False, False)
-    if len(out) < 2:
+
+    try:
```

Comment:
No except block? where are you catching this exception? or no need to catch this?

Reply:
yes no need to catch, we will raise exception so that it fails when required.